src/cat_usrdata.h

```diff
@@ -0,0 +1,67 @@
+/****************************************************************************
+ * src/cat_usrdata.h
+ *
+ * Copyright (C) 2021 Xiaomi Corperation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ****************************************************************************/
+#ifndef __CAT_USRDATA_H
+#define __CAT_USRDATA_H
+
+/*user define*/
+#define WRITEBUF_LEN        256
+#define READBUF_LEN         WRITEBUF_LEN
+#define ACKBUF_LEN          512
```

Comment:
hardcoded buffers sizes should be strongly avoded (please use static descriptors instead)